data/human/intro missions.txt

```diff
@@ -413,7 +413,91 @@ mission "Intro [2 Interceptor]"
 				`	"Sure, but I can't wait until I've got a ship big enough to stand up to pirates instead of running from them."`
 				`	"Well, maybe I should've bought a freighter instead."`
 			`	"Better start saving up then," he says. "I know a number of old Sparrow pilots who made their first million credits mining asteroids before they could afford a more combat-worthy ship. They had to look around for a while before finding a system with any asteroids worth mining, but it was worth it. Anyway, let's go get your payment." You walk out of the ship and sign the papers to hand off the <commodity> to the dock workers on <planet>. They hand you a credit chip worth <payment>.`
-			`	"If you're willing to take me on one more journey," says James, "meet me in the spaceport bar in a little while."`
+			`	"I've got an idea for something you might want to try," says James. "Meet me in the spaceport bar in a little while."`
+
+
+
+mission "Intro [2.1 Interceptor]"
+	priority
+	name "Mine asteroids in <system>"
+	description "Go to the <system> system and mine 5 tons of Silicon, then land on <planet>."
+	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
+	passengers 1
+	source "Ingot"
+	destination "New Sahara"
+	to offer
+		has "Intro [2 Interceptor]: done"
+		not "Intro [3 Interceptor]: offered"
+	
+	on offer
+		conversation
+			`James waves you over to his table when you enter the bar. He's sitting with another man with a local star map spread out between them. There's a steady patter of micrometeorites against the roof that reminds you of rain on the factory roof in New Boston.`
+			`	"This is Captain Marshall," James says as you pull up a chair. "I told him about your new Sparrow and how it'd be a good idea for you to get a taste of mining. He mines in this part of space and says there's a good system nearby if you're willing to give it a try."`
+			choice
+				`	"I'm willing to give it a shot."`
+					goto mining
+				`	"I'm ready to head out on my own."`
+					decline
```

Comment:
```suggestion
			`You find James waiting for you just outside the spaceport bar. Against the steady patter of micrometeorites against the roof he tells you, "Working here is one of the few jobs in the galaxy more dangerous than fighting pirates. And worse, because if a big meteorite decides to strike this station, there's basically nothing they can do about it except hope they see it coming." He turns to you with a grin. "But nonetheless, I think you might be interested in this idea."`
			`	James leads you down the side of the bar to a booth where a man with a half-eaten hamburger and a plate of fries is sitting, busily examining a star map of the local sector. He notices your arrival and stands up to greet you. "<first>, this is Captain Marshall," James says as you shake his hand. "I told him about your new ship and how it'd be a good idea for you to get a taste of asteroid mining. He mines in this part of space and says there's a good system nearby if you're willing to give it a try."`
			choice
				`	"I'll give it a shot."`
					goto mining
				`	"I think I'm ready to head out on my own."`
					decline
```
I miss this little tidbit, and also it lines up with the blocked message if James meets you and then walks into the bar to greet Marshall.